pytensor/link/pytorch/dispatch/basic.py

```diff
@@ -132,3 +133,16 @@ def makevector(*x):
         return torch.tensor(x, dtype=torch_dtype)
 
     return makevector
+
+
+@pytorch_funcify.register(IfElse)
+def pytorch_funcify_IfElse(op, **kwargs):
+    n_outs = op.n_outs
+
+    def ifelse(cond, ifpath, elsepath, n_outs=n_outs):
+        if cond:
+            return ifpath
+        else:
+            return elsepath
```

Comment:
No need for `n_outs` kwarg. 

More importantly you need to use `n_outs` to decide which inputs to return, since the signature is simply `ifelse(cond, *true_and_false_outputs)`.

That is, there can be multiple true and false outputs (the same number though), and you must return the first ha!f if the condition is true or the second half otherwise